website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -205,38 +205,26 @@ we will need consider later include items like updating the latest flag in the
 database and glacier on a file version when doing a recovery.
 
 ```json
-[
+"anotherPage": False,
+"granules": [
   {
-    "providerId": "lpdaac",
-    "collections": [
+    "provider_id": "lpdaac",
+    "collection_id": "MOD14A1___061",
+    "id": "MOD14A1.061.A23V45.2020235",
+    "createdAt": "2020-01-01T23:00:00Z",
+    "executionId": "u654-123-Yx679",
+    "ingestDate": "2020-01-01T23:00:00Z",
+    "lastUpdate": "2020-01-01T23:00:00Z",
+    "files": [
       {
-        "collectionId": "MOD14A1__061",
-        "shortname": "AST_L1A",
-        "collectionVersion": "001",
-        "granules": [
-          {
-            "granuleId": "MOD14A1.061.A23V45.2020235",
-            "cumulusCreateTime": "2020-01-01T23:00:00+00:00",
-            "executionId": "u654-123-Yx679",
-            "ingestDate": "2020-01-01T23:00:00+00:00",
-            "lastUpdate": "2020-01-01T23:00:00+00:00",
-            "files": [
-              {
-                "fileName": "MOD14A1.061.A23V45.2020235.2020240145621.hdf",
-                "cumulusArchiveLocation": "s3://cumulus-archive/MOD14A1/061/...",
-                "orcaLocation": "s3://orca-archive/MOD14A1/061/...",
-                "keyPath": "MOD14A1/061/032/MOD14A1.061.A23V45.2020235.2020240145621.hdf",
-                "fileSizeBytes": 100934568723,
-                "fileHash": "ACFH325128030192834127347",
-                "fileHashType": "SHA-256",
-                "fileVersion": "VXCDEG902",
-                "orcaEtag": "YXC432BGT789"
-              },
-              ...
-            ]
-          },
-          ...
-        ]
+        "name": "MOD14A1.061.A23V45.2020235.2020240145621.hdf",
+        "cumulusArchiveLocation": "cumulus-bucket",
+        "orcaArchiveLocation": "orca-archive",
+        "keyPath": "MOD14A1/061/032/MOD14A1.061.A23V45.2020235.2020240145621.hdf",
+        "sizeBytes": 100934568723,
+        "hash": "ACFH325128030192834127347",
+        "hashType": "SHA-256",
+        "version": "VXCDEG902"
```

Comment:
I will make the same changes to the SQS body.json according to this then

Reply:
@rizbi-hassan are you are referring to naming convention not structure correct?

Reply:
yes the name of keys like `sizeBytes`, `createdAt` will get changed. The schema structure will remain same, right? @bhazuka